src/main/java/com/pinterest/secor/io/impl/JsonORCFileReaderWriterFactory.java

```diff
@@ -181,7 +181,7 @@ private CompressionKind resolveCompression(CompressionCodec codec) {
             return CompressionKind.LZ4;
         else if (codec instanceof SnappyCodec)
             return CompressionKind.SNAPPY;
-        else if (codec instanceof ZlibCodec)
+        else if (codec instanceof GzipCodec)
             return CompressionKind.ZLIB;
```

Comment:
Looks fine.  How about you add a comment here to explain why we would need to use GzipCodec to trigger ZLIB compression kind, then we are good to go.